spec/services/iaa_reporting_helper_spec.rb

```diff
@@ -37,6 +44,13 @@
     build_integration(issuer: iaa2_sp.issuer, partner_account: partner_account2)
   end
 
+  let(:integration3) do
+    build_integration(partner_account: partner_account1, service_provider: service_provider1)
+  end
+  let(:integration4) do
+    build_integration(partner_account: partner_account2, service_provider: service_provider2)
+  end
```

Comment:
looks like we override these inside the context for `#partneraccounts`, so we can remove these because they're unused?
```suggestion
```